src/components/events/EventPosterUpload.tsx

```
@@ -34,12 +41,35 @@ export const EventPosterUpload: React.FC<EventPosterUploadProps> = ({ eventId, o
         }
 
         setPreviewUrl(URL.createObjectURL(file));
-        const success = await uploadAndProcess(eventId, file);
+
+        // Accessibility Linter: check contrast + alt text before publishing
+        const imageData = await getImageDataFromFile(file);
```

ESLint

[error] Delete `····`

(prettier/prettier)